src/modules/auth/tests/auth.service.spec.ts

```diff
@@ -334,11 +334,10 @@ describe('AuthenticationService', () => {
     const emailData = {
       to: email,
       subject: 'Reset Password',
-      template: 'reset-password',
+      template: 'Password-Reset-Complete-Template',
       context: {
-        link: 'http://example.com/auth/reset-password',
-        email: email,
-        token: '123456',
+        recoveryLink: 'http://example.com/auth/reset-password',
```

Comment:
Remove the link from the context, only the email and token is required. Leaving it this way will introduce breaking change